From 09385d5912306ccf444d4532520d339774fea48a Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Tue, 29 Dec 2015 17:54:31 -0500
Subject: [PATCH] sys/types.h: drop sys/sysmacros.h include

We want to break apart this include path due to namespace pollution.
https://sourceware.org/ml/libc-alpha/2015-11/msg00253.html
---
 posix/sys/types.h | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/posix/sys/types.h b/posix/sys/types.h
index b3a27c51f0e6..788364a299c9 100644
--- a/posix/sys/types.h
+++ b/posix/sys/types.h
@@ -216,15 +216,6 @@ typedef int register_t __attribute__ ((__mode__ (__word__)));
 
 /* It also defines `fd_set' and the FD_* macros for `select'.  */
 # include <sys/select.h>
-
-/* BSD defines `major', `minor', and `makedev' in this header.
-   However, these symbols are likely to collide with user code, so we are
-   going to stop defining them here in an upcoming release.  Code that needs
-   these macros should include <sys/sysmacros.h> directly.  Code that does
-   not need these macros should #undef them after including this header.  */
-# define __SYSMACROS_DEPRECATED_INCLUSION
-# include <sys/sysmacros.h>
-# undef __SYSMACROS_DEPRECATED_INCLUSION
 #endif /* Use misc.  */
 
 
-- 
2.11.0

